### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by naruto-4**

### CODE REVIEW

**Comment by naruto-4** [comment]
is there anything else i could add?

File: README.md [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/magento/magento2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/magento-2/localized.svg)](https://crowdin.com/project/magento-2)
-<h2>Welcome</h2>
-Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a cutting-edge, feature-rich eCommerce solution that gets results.
+
+## What is Magento 2?
+It's an e-commerce platform written in PHP that provides online merchants with a flexible shopping cart system as well as control over the look, functionality, and content of the online store. It also offers powerful search engines for marketing and catalog management tools. 
+
+**Welcome** to your magento 2 installation ! We're glad you chose to install Magento 2.
```
Comment (ptylek) [inline README.md:10]: Unnecessary space after `installation`

File: README.md [hunk 2]
```diff
@@ -1,19 +1,24 @@
 [![Open Source Helpers](https://www.codetriage.com/magento/magento2/badges/users.svg)](https://www.codetriage.com/magento/magento2)
+
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/magento/magento2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
 [![Crowdin](https://d322cqt584bo4o.cloudfront.net/magento-2/localized.svg)](https://crowdin.com/project/magento-2)
-<h2>Welcome</h2>
-Welcome to Magento 2 installation! We're glad you chose to install Magento 2, a cutting-edge, feature-rich eCommerce solution that gets results.
+
+## What is Magento 2?
+It's an e-commerce platform written in PHP that provides online merchants with a flexible shopping cart system as well as control over the look, functionality, and content of the online store. It also offers powerful search engines for marketing and catalog management tools.
```
Comment (ptylek) [inline README.md:8]: I think that last sentence shouldn't be `search engines for marketing` but `marketing, search engine optimization, and catalog-management tools`. Also why would you change that paragraph from `Welcome` to `What is Magento 2?`?

File: README.md (hunk 1 above)
Comment (ptylek) [reply README.md:10]: `Magento` should use capital letter

**Review by ptylek — CHANGES_REQUESTED** [review]
I am not sure about changing first paragraph. Why did you make this change? Also the text contains errors. Requested changes.

**Comment by ptylek** [comment]
@missione , I am closing this PR now due to inactivity. Also format is not valid .
Please reopen and update if you wish to continue.
Thank you for collaboration

**Pull request closed without merging**